clang/lib/3C/Constraints.cpp

```diff
@@ -440,7 +441,7 @@ bool Constraints::graphBasedSolve() {
       // 1. Find return vars with a lower bound.
       std::set<VarAtom *> ParamVars = Env.filterAtoms(IsParam);
       std::set<VarAtom *> LowerBoundedRet =
-          findBounded(SolPtrTypCG, &ParamVars, true);
+          findBounded(SolPtrTypCG, &ParamVars, true, true);
```

Comment:
Revert to the original (default for `useConstAtoms` is true already). Same with the one below.